### PR TITLE
[Spotless] Applying Google Code Format for core/src/main files #3

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description

This PR applies Spotless on Java code for:
`core/src/main/java/org/opensearch/sql/planner`
`core/src/main/java/org/opensearch/sql/storage`
`core/src/main/java/org/opensearch/sql/utils` 

 
### Issues Resolved
[[List any issues this PR will resolve]](https://github.com/opensearch-project/sql/issues/1101)
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by GumpacG** [comment]
Checkstyle fails for some of the changes.

**Comment by MitchellGale** [comment]
> Checkstyle fails for some of the changes.

Yeah, sadly I think I'll have to disable checkstyle. I'm looking into making Spotless have the same rules as checkstyles to see if that works.

File: core/src/main/java/org/opensearch/sql/planner/logical/LogicalValues.java [hunk 1]
```diff
@@ -14,16 +13,14 @@
 import org.opensearch.sql.expression.LiteralExpression;
 
 /**
- * Logical operator which is a sequence of literal rows (like a relation).
- * Basically, Values operator is used to create rows of constant literals
- * "out of nothing" which is corresponding with VALUES clause in SQL.
- * Mostly all rows must have the same number of literals and each column should
- * have same type or can be converted implicitly.
- * In particular, typical use cases include:
```
Comment (acarbonetto) [inline core/src/main/java/org/opensearch/sql/planner/logical/LogicalValues.java:22]: can we ignore spotless here?
or add list tags...

File: core/src/main/java/org/opensearch/sql/planner/optimizer/LogicalPlanOptimizer.java [hunk 2]
```diff
@@ -20,56 +19,48 @@
 import org.opensearch.sql.planner.optimizer.rule.write.CreateTableWriteBuilder;
 
 /**
- * {@link LogicalPlan} Optimizer.
- * The Optimizer will run in the TopDown manner.
- * 1> Optimize the current node with all the rules.
- * 2> Optimize the all the child nodes with all the rules.
- * 3) In case the child node could change, Optimize the current node again.
+ * {@link LogicalPlan} Optimizer. The Optimizer will run in the TopDown manner. 1> Optimize the
```
Comment (acarbonetto) [inline core/src/main/java/org/opensearch/sql/planner/optimizer/LogicalPlanOptimizer.java:22]: can we ignore spotless here or add list tags?

**Review by acarbonetto — APPROVED** [review]

File: core/src/main/java/org/opensearch/sql/planner/logical/LogicalValues.java (hunk 1 above)
Comment (MitchellGale) [reply core/src/main/java/org/opensearch/sql/planner/logical/LogicalValues.java:22]: Fixed

File: core/src/main/java/org/opensearch/sql/planner/physical/NestedOperator.java [hunk 3]
```diff
@@ -128,49 +117,38 @@ public ExprValue next() {
...
-   * and the Object it refers to the value.
+   * Simplifies the structure of row's source Map by flattening it, making the full path of an
+   * object the key and the Object it refers to the value.
    *
-   * <p>Sample input:
-   * keys = ['comments.likes']
-   * row = comments: {
-   * likes: 2
-   * }
+   * <p>Sample input: keys = ['comments.likes'] row = comments: { likes: 2 }
```
Comment (Yury-Fridlyand) [inline core/src/main/java/org/opensearch/sql/planner/physical/NestedOperator.java:141]: Add `pre` or few `br`

**Review by Yury-Fridlyand — APPROVED** [review]

File: core/src/main/java/org/opensearch/sql/planner/physical/NestedOperator.java (hunk 3 above)
Comment (MitchellGale) [reply core/src/main/java/org/opensearch/sql/planner/physical/NestedOperator.java]: Thanks! added `pre` in d52b65cd9fbbf4aa5599a663c3872f896f27daca

**Pull request merged**